core/domain/subtopic_validators_test.py

```diff
@@ -98,6 +98,10 @@ def setUp(self):
         self.model_instance_2 = (
             subtopic_models.SubtopicPageModel.get_by_id('2-1'))
 
+        self.job_class = (
+            prod_validation_jobs_one_off
+            .SubtopicPageSnapshotContentModelAuditOneOffJob)
```

Comment:
But these are not tests for `SubtopicPageSnapshotContentModel` but for `SubtopicPageModel`, ditto in other places.

Reply:
I though they are different